utils/scraper_processing_utils.R

```diff
@@ -186,13 +186,19 @@ format_name_str <- function(in_str){
 #' @param key_str, a query string 
 #' @param str_vec, a string vector
 #' @return string, a string from str_vec that has closest distance to key_str
-get_matched_string <- function(key_str, str_vec){
+get_matched_string <- function(key_str, str_vec, max_cost=NA){
```

Comment:
Is this function related to any existing distance function, such as  the [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance)? I think `adist` is related to it. Maybe it would be good to add that to the documentation.

Reply:
Yes, exactly! adist is like a generalized Levenshtein distance, but with weighted cost. Here I have a cost for deletion =0. This is a good point to add more of my reasoning here; I added more explanation to this in the documentation.
added here: https://github.com/greenelab/nature_news_disparities/pull/23/commits/ff2aad88a1f79c9a3a68274ad5aba532313cdf86

Reply:
For easier reference, this is what I added:
> #' Given a query string, return closest string in the
#' vector of reference strings (no penalty for deletion)
#' using adist: "the generalized Levenshtein (edit) distance, 
#' giving the minimal possibly weighted number of insertions, 
#' deletions and substitutions needed to transform one string into another.
#' For our use, deletions are NOT penalized and have weight 0
#' this is because when matching names, we may only get a last name 
#' and would like to match it to a full name. 
#' Example query string: "John"
#' reference strings: "John Fillipio", "Jane Do"
#' without deletion weight of 0, John has the closest distance to "Jane Doe"